x-pack/plugins/security_solution/public/common/components/discover_in_timeline/use_discover_in_timeline_actions.tsx

```diff
@@ -69,18 +73,23 @@ export const useDiscoverInTimelineActions = (
     },
   });
 
-  const defaultDiscoverAppState: DiscoverAppState = useMemo(() => {
+  const getDefaultDiscoverAppState: () => Promise<DiscoverAppState> = useCallback(async () => {
+    const localDataViewId = dataViewId ?? 'security-solution-default';
+
+    const dataView = await dataViewService.get(localDataViewId);
+
     return {
-      query: discoverDataService.query.queryString.getDefaultQuery(),
+      query: {
+        esql: `from ${dataView?.getIndexPattern()} | limit 10`,
```

Comment:
if data view is not defined - esql query would look like `from undefined | limit 10`
Is this possible for `dataView` to be undefined? Looks like returned type is always `DataView`, but optional chaining implies, it can be `undefined`
Can we use here default Security Solution indices or just set is as an empty string if dataview is not defined?